test/incrParse/add-space-at-missing-brace.swift

```diff
@@ -0,0 +1,7 @@
+class AnimationType {
```

Comment:
You are missing the `RUN` line for this test case.

Reply:
Well, this change doesn't fix this test case. Sorry, I will retry fixing.